examples/basic/package.json

```diff
@@ -15,5 +15,6 @@
     "eslint-config-custom": "*",
     "prettier": "^2.5.1",
     "turbo": "latest"
-  }
+  },
+  "packageManager": "^yarn@1.22.17"
```

Comment:
What does that caret do? I don't see it in the docs https://nodejs.org/api/corepack.html